cypress/features/regression/test/loader.feature

```diff
@@ -12,8 +12,8 @@ Feature: Loader default component
       | large   | 20     | 8      | sizeLarge    |
 
   @positive
-  Scenario Outline: Verify size of button with <size> loader
```

Comment:
Size seems to have been removed from here but it then won't specify the size when each example runs.